### ISSUE TITLE
Support direct ES6 module loading

### ISSUE THREAD

**Issue by FostUK** [lancedikson/bowser#292]
Our use case for a browser detect module uses
* Direct ES6 module loading for development (no build step on modern browsers)
* A final webpack build.

I have a local version of bowser which makes a few changes need to allow a direct import of bowser.js like:

```Javascript
import Bowser from "../../node_modules/bowser/src/bowser.js";
```
The changes needed are:
* Add file extension _.js_ to all imports
* Add an export to utils.js and import the whole from there.

Happy to do a PR for this you could look at but thought I'd ask first if this is somehting you are interested in?

**Comment by lancedikson** [comment]
Hi, @FostUK. I don't think that I clearly understood your case. I believe, you can have direct ES6 module loading using `import Bowser from 'bowser/src/bowser';` and get the source file loaded like that for development. 
For the final build I suggest allowing the build packing system you use (webpack, rollup, etc) to use transpiling (babel) on `/node_modules/bowser/src/*` and then it will process the source files into the final build as it should.
If I understood smth wrong, please, let me more details on the usage and we could come up with a solution :)

**Comment by FostUK** [comment]
Hi Lance!
When I say "direct" I mean - with no build steps whatsoever using a current browser such as chrome.
your example:
```Javascript
import Bowser from 'bowser/src/bowser'
```
Won't work for the simple reason that it doesn't have the file extension. The following works:
```Javascript
import Bowser from 'bowser/src/bowser.js'
```
Also your export in utils.js uses common.js style rather than ES6 style.
I believe direct loading can be supported with these minor changes and without breaking anything.
Totally happy to do that work in a PR for you to look at but wanted to check with you first.

Basically the above allows to do really fast development (Chrome loads the modules without needing any transpilation) and only run to run build step when we need to make a release.

**Comment by lancedikson** [comment]
Oh, right, I see. Yes, you're welcome to open a PR, I can go through it and merge :)

**Comment by FostUK** [comment]
Cool. I'll get somehting together asap.

**Comment by FostUK** [comment]
PR: #294 

Let me know if you need anything else or there are any issues.

**Comment by lancedikson** [comment]
Thanks @FostUK for your help. This one can be closed.
